src/components/GameItem.js

```diff
@@ -2,28 +2,57 @@ import React, { Component } from 'react';
 import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
+// import App from '../src/App.js'
+
 
 class GameItem extends Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      _checked: false,
+      _divClass: "game-item"
+    }
+  }
+
+  changedToCheck = () => {
+    if (!this.state.checked){
+
```

Comment:
This reads `state.checked`, but in the rest of your code you're looking at `_checked`. That means that this code will always run, and you can continue to click on already-clicked items and get more points.